front-end/src/app/reports/f3x/create-workflow/create-f3x-step2.component.html

```diff
@@ -0,0 +1,149 @@
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="field">
+            <label for="state">STATE/PROVINCE</label>
+            <p-dropdown
+              inputId="state"
+              formControlName="state"
+              [options]="stateOptions"
+              optionLabel="name"
+              optionValue="code"
+              [autoDisplayFirst]="false"
+              appendTo="body"
+            ></p-dropdown>
+            <app-error-messages [form]="form" fieldName="state" [formSubmitted]="formSubmitted"></app-error-messages>
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="field">
+            <label for="zip">ZIP/POSTAL CODE</label>
+            <input type="text" pInputText id="zip" formControlName="zip" />
+            <app-error-messages [form]="form" fieldName="zip" [formSubmitted]="formSubmitted"></app-error-messages>
+          </div>
+        </div>
+      </div>
+    </ng-container>
+    <p-divider></p-divider>
+    <div class="grid">
+      <div class="col-12">
+        <h3>Committee status</h3>
+        <div>
+          This committee has
+          <ng-container *ngIf="(committeeAccount$ | async)?.committee_type === 'N'"
+            ><span class="bold">NOT</span></ng-container
+          >
+          qualified as a <span class="bold">multicandidate committee</span>.
+        </div>
+        <div>
+          <a
+            href="https://www.fec.gov/help-candidates-and-committees/filing-pac-reports/multicandidate-status/"
+            target="_blank"
+            rel="noopener"
+            >Learn more about multicandidate committee status</a
+          >
+        </div>
+      </div>
+    </div>
+    <p-divider></p-divider>
+    <div class="grid">
+      <div class="col-12">
+        <div class="field">
+          <h3>Memo</h3>
+          <p-checkbox
+            [binary]="true"
+            formControlName="memo_checkbox"
+            label="I WOULD LIKE TO ENTER A MEMO RELATED TO THIS FORM 3X"
+            class="bold"
+            ariaLabel="I would like to enter a memo related to this Form 3X"
+          ></p-checkbox>
+          <div class="memo-checkbox-note">This step is optional and can be entered later in the filing process.</div>
+        </div>
+      </div>
+    </div>
+    <ng-container *ngIf="form.get('memo_checkbox')?.value">
+      <div class="grid">
+        <div class="col-12">
+          <div><span class="bold">REPORT LEVEL MEMO</span> (OPTIONAL)</div>
+          <div>
+            <textarea [rows]="5" [cols]="30" pInputTextarea [autoResize]="true" formControlName="memo"></textarea>
+          </div>
+        </div>
+      </div>
+    </ng-container>
+    <p-divider></p-divider>
+    <div class="grid">
+      <div class="col-2">
+        <button pButton pRipple label="Back" icon="pi pi-times" class="p-button-info" (click)="save('back')"></button>
```

Comment:
Do we want back to save?

Reply:
I ran it by Mitchell and he didn't say no. I assumed we want to save whenever we leave the page. A good question for Aurelia